homeassistant/components/recorder/websocket_api.py

```diff
@@ -199,6 +199,12 @@ async def ws_handle_get_statistics_during_period(
     else:
         end_time = None
 
+    if not (statistic_ids := msg["statistic_ids"]):
```

Comment:
Is this check needed? This method seems to be only called from `ws_get_statistics_during_period` which has new validation?

Reply:
Removed and switched to vol.Length in the schema 8d3fea260d1067a9741560abf8687122a027a3de